src/Svg.ts

```diff
@@ -177,23 +187,23 @@ export class SVGDrawing {
                     break;
                 case QRCodeFrame.BANNER_TOP:
                     this.shiftX = 1.5 * this.config.moduleSize;
-                    this.shiftY = 1.5 * this.config.moduleSize + size / 5 - 1;
+                    this.shiftY = (1.5 * this.config.moduleSize + size / 5 - 1) + multiLineHeight;
                     break;
                 case QRCodeFrame.BANNER_BOTTOM:
                     this.shiftX = 1.5 * this.config.moduleSize;
                     this.shiftY = 1.5 * this.config.moduleSize;
```

Comment:
why are we not adding multiLineHeight here, I am not aware of the the code but is this intended or we missed ?